blog/electron-27-0.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Electron 27.0.0
+date: 2023-10-10T00:00:00.000Z
+authors:
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: electron-27-0
+---
+
+Electron 27.0.0 has been released! It includes upgrades to Chromium `118.0.5993.32`, V8 `11.8`, and Node.js `18.17.1`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 27.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on Twitter, or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Stack Changes
+
+- Chromium `118.0.5993.32`
+  - [New in Chrome 118](https://developer.chrome.com/blog/new-in-chrome-118/)
+  - [New in DevTools 118](https://developer.chrome.com/blog/new-in-devtools-118/)
+- Node.js `18.17.1`
+  - [Node 18.17.1 blog post](https://nodejs.org/en/blog/release/v18.17.1/)
```

Comment:
Potentially worth including any other Node.js patches or minors since 18.6.1, which is what was included in 26.0.0. IMO at the very least, 18.17.0 would probably be a more interesting link than 18.17.1.

Reply:
Good point, added the 18.17.0 post - looks like there weren't any patches between 18.16.1 and 18.17.0, so that was easy 😄 We can include patches & minor in the future too, it's a good idea